test/unit/cameras/PerspectiveCamera.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @author simonThiele / https://github.com/simonThiele
+ */
+
+module( "PerspectiveCamera" );
+
+test( "updateProjectionMatrix", function() {
+	var near = 1,
+			far = 3,
+			bottom = -1,
+			top = 1,
```

Comment:
In the next update I use a reference matrix, calculated with plain WebGL and use it for comparing.